include/linux/rtnetlink.h

```diff
@@ -172,7 +186,7 @@ enum {
 #define RTM_NR_FAMILIES	(RTM_NR_MSGTYPES >> 2)
 #define RTM_FAM(cmd)	(((cmd) - RTM_BASE) >> 2)
 
-/*
+/* 
```

Comment:
looks like a number of white-space changes - including several trailing spaces like this?

Reply:
rtnetlink.h must match what the kernel has.  I have absolutely nothing to do with the indentation levels.  I merely typed `cp <from/the/kernel> include/linux`.

Reply:
ah, I wasn't aware that we were just copying that into place.